packages/meditrak-app/appcenter-post-clone.sh

```diff
@@ -1,31 +1,29 @@
 #!/usr/bin/env bash
 
-## This file runs on appcenter automated builds
+## This file runs on App Center automated builds
 
-## Move environment variables saved by appcenter as USER-DEFINED_VARIABLE_NAME into .env, ready for react-native-dotenv
-echo "Setting up environment variables"
+## Move environment variables saved by App Center as USER-DEFINED_VARIABLE_NAME into .env, ready for react-native-dotenv
+echo -e "\033[32mSetting up environment variables\033m"
 env | grep "USER-DEFINED_.*" | awk -F "USER-DEFINED_" '{print $2}' > .env
 
-# install nvm, yarn
+# Install nvm, Yarn
 curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
 export NVM_DIR="$HOME/.nvm"
 [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
 
-# Workaround for meditrak-app needing node v16 but the monorepo using v14
-echo "16.20.2" > ../../.nvmrc
```

Comment:
This is the only actual change

The rest is capitalisation and `echo`-ing something in 🟢green🟢